src/acceptance-test/java/net/consensys/orion/acceptance/MultiKeyOperationAcceptanceTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package net.consensys.orion.acceptance;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.concurrent.TimeUnit;
+import net.consensys.orion.acceptance.dsl.AcceptanceTestBase;
+import net.consensys.orion.acceptance.dsl.OrionNode;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import org.awaitility.Awaitility;
+import org.junit.Test;
+
+public class MultiKeyOperationAcceptanceTest extends AcceptanceTestBase {
+  private final byte[] payload = "This Is My Data".getBytes(StandardCharsets.UTF_8);
+
+  @Test
+  public void dataTransfersFromOneNodeToAnother() throws IOException, InterruptedException {
+    final OrionNode bootnode = orionFactory().create("node_1", 1);
+    bootnode.start();
+
+    final OrionNode secondNode = orionFactory().create("node_2", 1, singletonList(bootnode));
+    secondNode.start();
+
+    Awaitility.waitAtMost(5, TimeUnit.SECONDS).until(() -> bootnode.peerCount() > 1);
```

Comment:
As lines 33-39 are likely to be repeated in most tests, refactoring into either the base or a class similar to the `Cluster` from Pantheon might an idea.